src/main/java/com/google/cloud/genomics/dataflow/utils/VariantUtils.java

```diff
@@ -28,7 +28,10 @@
 
   public static boolean isVariant(Variant variant) {
     List<String> alternateBases = variant.getAlternateBases();
-    return !(null == alternateBases || alternateBases.isEmpty());
+    // The same deletion can be specified as [CAG -> C] or [AG -> null], so double check that the
+    // reference bases are also of length 1 when there are no alternates.
+    return !(LENGTH_IS_1.apply(variant.getReferenceBases()) 
+        && (null == alternateBases || alternateBases.isEmpty()));
```

Comment:
What about variants that are specified as [G -> C] ? Those would return false here but those are actually variants. Perhaps the test should be length(referenceBases) == length(alternateBases) ?